utility/src/main/java/org/oppia/android/util/parser/html/ListItemLeadingMarginSpan.kt

```diff
@@ -88,7 +85,7 @@ sealed class ListItemLeadingMarginSpan : LeadingMarginSpan {
   class OlSpan(
     context: Context,
     private val indentation: Int,
-    private val contents: String
+    private val bulletNumbers: String
```

Comment:
Isn't this only for numbered text items? Perhaps ``numberedItemPrefix`` might be a bit clearer?

Reply:
Done.